package.json

```diff
@@ -82,7 +83,8 @@
   },
   "scripts": {
     "test": "jest test/*.spec.js --verbose -i",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "postversion": "mdchangelog --remote stardog-union/stardog.js --no-prologue --order-milestones semver --order-issues closed_at --overwrite --no-orphan-issues"
```

Comment:
So let's make this `version` so that the changes and changelog all go together with the same commit.